render/detailed_node_test.go

```diff
@@ -1,4 +1,4 @@
-package report_test
+package render
```

Comment:
Please keep this in the _test package. The only reason _test.go files should be in the original package namespace is if they're explicitly testing package/component internals, and even then it's a bad smell unless the package/component is really complex.